packages/oasis-actions/src/strategies/aave/adjust.ts

```diff
@@ -110,7 +128,7 @@ export async function adjust(
         oazo: new BigNumber(FEE),
       },
       prices: {
-        market: quoteMarketPrice,
+        market: isIncreasingRisk ? quoteMarketPrice : ONE.div(quoteMarketPrice),
```

Comment:
I'm not sure that inverse of the quote market price is right. Image eth -> steth, you can swap 1:1 eth to stEth but stEth -> eth is usually less then 1.